### PR TITLE
feat: add popsicle finance univ3 lp agg exploit

### PR DESCRIPTION
**Pull request by akshaynexus**

### CODE REVIEW

**Comment by akshaynexus** [comment]
Anything remaining to merge this?

**Comment by leggod** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
liaobinbin, @langrenfengzi, @hyperse, @ericlqs, @AndyLee1024, @yuezhiniao, @linlexuan2005, @q32854978lhy, @Love4yzp, @303snowing, @h4m5t, @bingryan, @rich678s, @terryjianluo, @ChamnLee, @GeistSher, @peter-sai, @Holybasil, @mavisguan, @lzxry

**Comment by akshaynexus** [comment]
@SunWeb3Sec please check this and merge if possible thanks

File: foundry.toml [hunk 1]
```diff
@@ -3,7 +3,7 @@ src = 'src'
 out = 'out'
 libs = ['lib']
 
-rpc_endpoints = { mainnet = "https://rpc.ankr.com/eth", blast = "https://rpc.ankr.com/blast", optimism = "https://optimism.llamarpc.com", fantom = "https://rpc.ankr.com/fantom", arbitrum = "https://rpc.ankr.com/arbitrum", bsc = "https://rpc.ankr.com/bsc", moonriver = "https://moonriver.public.blastapi.io", gnosis = "https://rpc.ankr.com/gnosis", Avalanche = "https://rpc.ankr.com/avalanche", polygon = "https://rpc.ankr.com/polygon", celo = "https://rpc.ankr.com/celo", Base = "https://developer-access-mainnet.base.org" }
+rpc_endpoints = { mainnet = "https://eth-pokt.nodies.app", blast = "https://rpc.ankr.com/blast", optimism = "https://optimism.llamarpc.com", fantom = "https://rpc.ankr.com/fantom", arbitrum = "https://rpc.ankr.com/arbitrum", bsc = "https://rpc.ankr.com/bsc", moonriver = "https://moonriver.public.blastapi.io", gnosis = "https://rpc.ankr.com/gnosis", Avalanche = "https://rpc.ankr.com/avalanche", polygon = "https://rpc.ankr.com/polygon", celo = "https://rpc.ankr.com/celo", Base = "https://developer-access-mainnet.base.org" }
```
[the remark this answers is not in the document]
Comment (akshaynexus) [reply foundry.toml:6]: The older rpc when testing was returning not activated error,dont think thats the case now

**Comment by SunWeb3Sec** [comment]
I will close this PR. please submit after refactor the repo.

**Pull request closed without merging**